insights/components/satellite.py

```diff
@@ -91,3 +91,33 @@ class IsSatellite611(IsSatellite):
     """
     def __init__(self, sat):
         super(IsSatellite611, self).__init__(sat, 6, 11)
+
+
+@component(SatelliteVersion)
+class IsSatellite614AndLater(object):
+    """
+    This component uses ``SatelliteVersion`` combiner
+    to determine the Satellite version. It checks if the Satellite version is 6.14 and later,
+    and raises ``SkipComponent`` when it isn't.
+
+    Raises:
+        SkipComponent: When the Satellite version is earlier than 6.14.
+    """
+    def __init__(self, sat):
+        if sat.major == 6 and sat.minor < 14:
```

Comment:
Hi @huali027 , the code looks good to me. Jut one concern here. 

I am new to Satellite things, and just from the code, I guess that a "Satellite 5.x" will also hit this `IsSatellite614AndLater` component. 
Do we process only "Satellite 6" systems, or something are already be done to assured for this.

Reply:
@JoySnow I updated it to exclude satellite 5.x as discussed. 
@xiangce Please help to review too.

Reply:
Thanks @huali027 , the code looks good to me .